clinica/utils/input_files.py

```diff
@@ -457,20 +457,27 @@ def pet_volume_normalized_suvr_pet(
 # pet-linear
 
 
-def pet_linear_nii(acq_label, suvr_reference_region, uncropped_image):
+def pet_linear_nii(acq_label, suvr_reference_region, uncropped_image, use_uniform):
     import os
 
     if uncropped_image:
         description = ""
     else:
         description = "_desc-Crop"
 
+    if use_uniform:
+        pipeline_description = "Uniform PET data"
+        rec = "_rec-uniform"
+    else:
+        pipeline_description = ""
+        rec = ""
+
     information = {
         "pattern": os.path.join(
             "pet_linear",
-            f"*_trc-{acq_label}_pet_space-MNI152NLin2009cSym{description}_res-1x1x1_suvr-{suvr_reference_region}_pet.nii.gz",
+            f"*_trc-{acq_label}_pet_space-MNI152NLin2009cSym{description}_res-1x1x1_suvr-{suvr_reference_region}{rec}_pet.nii.gz",
```

Comment:
Better to keep source entities ordered the same way as the [BIDS entity table](https://bids-specification.readthedocs.io/en/stable/99-appendices/04-entity-table.html). 

```suggestion
            f"*_trc-{acq_label}{rec}_pet_space-MNI152NLin2009cSym{description}_res-1x1x1_suvr-{suvr_reference_region}_pet.nii.gz",
```